dependabot/resources/connector_list.json

```diff
@@ -1,9 +1,73 @@
 {
-    "auto_bump": false,
+    "auto_bump": true,
     "modules": [
         {
             "name": "module-ballerinax-slack",
-            "auto_merge": false
-        }
+            "auto_merge": true
+        },
+        {
+            "name": "module-ballerinax-github",
```

Comment:
Do all of these have Gradle files that need to be updated?

Reply:
@niveathika Yes. Each of these repos have gradle.properties file and the ballerinaLangVersion is there in that file